admin/app.js

```diff
@@ -73,7 +72,6 @@ const app = express()
 
 setupBrowserSecurity(app)
 setupLogging(app)
-// checkConfigWhitelist(app)
```

Comment:
Why is this removed?  Isn't it still required?

Reply:
@jon-shipley my understanding from @GuyHarwood is that this is no longer required.